modules/itemManager.ts

```diff
@@ -0,0 +1,156 @@
+import db, { Item } from './db';
+import {
+  renderItemsList,
+  renderSectionBubbles,
+  noItemsMessage,
+} from './domUtils';
+import { autoSort, hideChecked, sectionSort } from './domElements';
+
+interface SectionData {
+  total: number;
+  isPurchased: number;
+}
+
+type StoreSectionData = {
+  [key: string]: SectionData;
+};
+
+//TODO: we could consider further breaking this down, by adding an ItemsService for db interactions
```

Comment:
This could be an option to slim this class down and make it easier to read.... or we could also create a directory for the `ItemManager` and create one file per method. I think both have their pros and cons, but I try to keep PR's to 10 touched files or less, so we can do this later if reading this class becomes a real pain.